main.py

```diff
@@ -31,6 +31,7 @@
 
 
 app = Flask(__name__)
+app.logger.setLevel("INFO")
```

Comment:
Specify the log level in config.py? See https://docs.python.org/3/library/logging.html#logging-levels

E.g. LOG_LEVEL='DEBUG' in the DevConfig and TestConfig? 'INFO' in the prodconfig? 

Also, what are the implications of setting info vs warning? I'm not too familiar with Python logging.

Are there other settings that might be a good idea to set?

Reply:
I found that INFO essentially log "confirmations that things are working as expected" while WARNING logs "indications that something unexpected happened" which could potentially mean an error could occur. I think that it would be useful to make ProdConfig set the level to INFO since info that things are working as expected could be potentially useful to customer.

Looking more into loggers, I couldn't find any other settings that would be useful to set. However, I found that more handlers (which send a log to a specific destination) can be added to a logger if desired which could be useful to users who want to add additional logging destinations besides Google's logging or stdout.

Reply:
I see, so it sounds like it mainly depends on how different systems into INFO vs WARNING logs.

Reply:
Done